composer.json

```diff
@@ -18,8 +18,9 @@
         "php": ">=5.4.0 <8.0",
         "ext-json": "*",
         "ext-mbstring": "*",
-        "phpunit/phpunit": ">4.8.20 <6.0",
-        "phpunit/php-code-coverage": ">=2.2.4 <5.0",
+        "phpunit/phpunit": ">4.8.20 <6.1",
+        "phpunit/php-code-coverage": ">=2.2.4 < 4.1 || ^5.0",
+        "phpunit/phpunit-mock-objects": ">2.3 <3.5 || ^4.0",
```

Comment:
Same reason for this one. What would be your preferred set of values for these versions?